src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD105AvoidImplicitTaskSchedulerCurrentAnalyzerTests.cs

```diff
@@ -52,12 +42,17 @@ class Test {
 }
 ";
 
-            this.expect.Locations = new[] { new DiagnosticResultLocation("Test0.cs", 6, 22, 6, 30) };
-            this.VerifyCSharpDiagnostic(test, this.expect);
+            var expected = Verify.Diagnostic().WithSpan(6, 22, 6, 30);
+            await new Verify.Test
+            {
+                TestCode = test,
+                ExpectedDiagnostics = { expected },
+                VerifyExclusions = false,
+            }.RunAsync();
```

Comment:
What does `VerifyExclusions` mean? Is this test framework we're using documented somewhere? Or at least are there xml doc comments that would make this discoverable if I had this open in VS?